lblod/app-lokaal-mandatenbeheer
# add public election results

## Description

add public election results

## How to test

Run your migrations and verify that the new election results are visible in the frontend
